### PR TITLE
Added Support for browsers without grid support

### PR DESCRIPTION
**Pull request by leoncai1999**
Layout fixes include the following:

- More research cards on homepage
- About us page
- Staff page
- grid view of articles in verticals
- layout of individual articles

There are a few more visual fixes that can be made across the site for non-grid users, but we think this is a good checkpoint.

### CODE REVIEW

File: wp-content/themes/engage/assets/scss/components/_tiles.scss [hunk 1]
```diff
@@ -56,21 +73,33 @@
...
+		grid-row: 1;
+		padding: 4px 0px 2px 8px;
+		margin: 15px 0 9px;
+		//  border-left: 2px solid $teal;
+		font-size: 12px;
+		line-height: 1;
+		color: $gray-dark;
+	}
+	@supports not (display: grid) {
+		width: 0%;
```
Comment (luukee) [inline wp-content/themes/engage/assets/scss/components/_tiles.scss:101]: `0` is better than `0%` 
:)

**Review by luukee — APPROVED** [review]
Looks pretty good to me. Tomorrow I'll check it out on my local & let you know!

File: wp-content/themes/engage/assets/scss/templates/_homepage.scss [hunk 2]
```diff
@@ -19,6 +19,19 @@ $lightGrey: #fbfbfb;
 			grid-column-end: span 3;
 		}
 
+		@supports not (display: grid) {
+			display: flex;
+			flex: 0 1 23.6%;
+	
+			@media screen and (max-width: 1200px) {
```
Comment (luukee) [inline wp-content/themes/engage/assets/scss/templates/_homepage.scss:26]: @Lay-ton @NWebb5 One thing to notice is we should use the media query mixins (`@include media($mobile) {}` ext.) for media queries. I made a big comment in the `_mixins.scss` file of which I will be committing Monday :)

Comment (leoncai1999) [reply wp-content/themes/engage/assets/scss/templates/_homepage.scss:26]: Got it! I look forward to seeing that commit!

**Pull request merged**